table/command.go

```diff
@@ -102,7 +101,11 @@ func (o Options) Run() error {
 	}
 
 	m := tm.(model)
-	fmt.Println(strings.Join([]string(m.selected), o.Separator))
+
+	w := csv.NewWriter(os.Stdout)
+	w.Comma = reader.Comma
+	w.Write([]string(m.selected))
```

Comment:
We can probably just `return w.Write(...)` here. This also resolves the linter error.

Reply:
`Flush` needs to be called or it won't actually print.